index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+
+		<title>ToDo List</title>
+		<meta name="description" content="ToDo application" />
+		<meta name="author" content="Bartlomiej Zdybel" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+		<link rel="stylesheet" type="text/css" href="./index.css" />
+
+		<link rel="icon" href="icon_code.jpg" />
+	</head>
+
+	<body>
+		<div class="container">
+			<div class="task-list__container">
+				<form class="task-form task-form__add">
+					<h1 class="task-form__title">task list</h1>
+
+					<div class="task-form__input-wrapper">
+						<label class="task-form__input task-form__input--title" for="task">What must you do?</label>
```

Comment:
It's incorrect to assign the `task-form__input` class to labels - because they are labels, not inputs :P (even with --title modifier)